services/users/src/test/java/org/opfab/users/model/CurrentUserWithPerimetersDataShould.java

```diff
@@ -22,22 +22,28 @@ void testComputePerimeters(){
         Perimeter p1 = PerimeterData.builder().
                 id("perimeterKarate10_1_RR").
                 process("process10").
-                stateRights(new HashSet<>(Arrays.asList(new StateRightData("state1", RightsEnum.RECEIVE),
-                                                        new StateRightData("state2", RightsEnum.RECEIVE)))).
+                stateRights(new HashSet<>(Arrays.asList(new StateRightData("state1", RightsEnum.RECEIVE, true),
+                                                        new StateRightData("state2", RightsEnum.RECEIVE, true)))).
```

Comment:
Add in this class a test to check merging of filter notification allowed (testMergeNotificationAllowed)

Reply:
Done.